src/components/gameScene/animatedComp/AnimatedScrollForThemes.tsx

```diff
@@ -0,0 +1,65 @@
+import { RoundStateData } from "@app/SignalR/MessageTypes";
+import { Box, Typography } from "@mui/joy";
+import { keyframes } from "@mui/styled-engine";
+import { useEffect } from "react";
+
+type TestingPros = {
+  changeIdx: () => void
+  questions: RoundStateData['Questions']
+}
+
+const breatheAnimation = keyframes`
+ 0% {
+    top : 100%;
+  }
+ 100% {
+    top : -200%;
+  }
+`
+
+export default function AnimatedScrollForThemes(props: TestingPros) {
+
+  const themes: string[] = [];
+
+  for (const theme in props.questions) {
+    themes.push(theme)
+  }
+
+  useEffect(() => {
+    const t = setTimeout(() => {
+      props.changeIdx()
+    }, themes.length * 1500)
```

Comment:
Please avoid magic number

It's better to make code that documents itself, for example:
```
const speed = 100
const heightItem = 150
const durationPerItem = heightItem / speed;
const totalDurationMs = themes.length *  durationPerItem * 1000
```

Reply:
This allows a better understanding of what the parameters affect, and a better understanding of how to change them in case there are too many or too few themes.

Reply:
Got it